client/www/scripts/modules/manager/manager.controllers.js

```diff
@@ -40,7 +40,25 @@ Manager.controller('ManagerMainController', [
     $scope.toggleManagerLoadBalancer = function() {
       return $scope.showManagerLoadBalancer = !$scope.showManagerLoadBalancer
     };
-
+    $scope.loadBalancers = [];
+    $scope.loadLoadBalancers = function() {
+      $scope.loadBalancers = $scope.mesh.models.LoadBalancer.find({}, function(err, response) {
```

Comment:
@seanbrookes You shouldn't set `$scope.loadBalancers` to `LoadBalancer.find()` it doesn't return anything...